client/src/components/NavBar.tsx

```diff
@@ -0,0 +1,40 @@
+import { NavLink } from "react-router-dom";
+import Search from "./Search";
+
+function NavBar() {
+  return (
+    <>
+      <div className="navBarMW">
+        <img
+          className="logoMW"
+          src="../../public/Images/logo-musiwild-without-text.png"
+          alt="logo du site MusicWild"
+        />
+        <img
+          className="titleMW"
+          src="../../public/Images/MUSICWILD.png"
```

Comment:
public folder is accessible by your web app, you can update the import to "/Images/MUSICWILD.png"

Reply:
Dead code deleted